src/azure-cli/azure/cli/command_modules/monitor/_help.py

```diff
@@ -689,6 +689,11 @@
     text: |
         az monitor autoscale rule create -g {myrg} --autoscale-name {myvmss} \\
           --scale in 50% --condition "Percentage CPU < 25 avg 15m"
+  - name: Create autoscale settings to autoscale via a guest vm metric enabled from diagnostic extensions.
```

Comment:
>to autoscale [](start = 36, length = 12)

remove "to autoscale" as this command is in autoscale itself?

Reply:
fix